src/main/java/seedu/duke/card/Subject.java

```diff
@@ -9,27 +11,36 @@
     private String title;
     private TopicList topics;
     private TaskList tasks;
+    private List<Flashcard> flashcards;
```

Comment:
List of flashcards is not used, can be removed

Reply:
Noted